### DISCUSSION TITLE
Detect new tab

### DISCUSSION THREAD

**Discussion by SamuelTanielM** [AutomaApp/automa#264] [Q&A]
![image](https://user-images.githubusercontent.com/97520448/148933965-f1ded6d1-af43-41eb-b407-92fbfdcda625.png)
So the trigger event is used to click a submit button of a form, and if successful (the website sometimes doesn't let me go through to the next page so I have to make a workflow to click the submit button a lot of times until it works) a new tab will show up, what function should I use to detect a new tab so that after a new tab comes up from the form website, I could use another flow like opening a youtube web to make a sound that says "hey you did it!"

**Comment by Kholid060** [comment]
Currently, Automa doesn't have a feature to detect a new tab.
